Aptoide/aptoide-client-v8
# [Feature/ASV-787] Appc in appview

**What does this PR do?**

   Shows appc in-app-billing information in appview if app supports and it doesn't have advertising flow.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] NewAppViewFragment.java

**How should this be manually tested?**

   Search for "2048 asf" > click search result > app view should show that app supports in-app billing. 

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-787](https://aptoide.atlassian.net/browse/ASV-787)

**Code Review Checklist**
